packages/core/src/components/panel-stack/_panel-stack.scss

```diff
@@ -59,6 +59,10 @@
   .#{$ns}-dark & {
     background-color: $dark-gray4;
   }
+
+  &:nth-last-child(n + 4) {
```

Comment:
_All but last 3_. Why 3? When one removes the top panel, the bottom two should be visible to animate properly. The rest can be hidden as a layering optimization.

Reply:
> The rest can be hidden as a layering optimization.

Can you provide any evidence that we actually need this optimization?

Reply:
Well, it's not really _needed_ - I wanted to make it in align with the `Tabs` implementation ([JSX](https://github.com/palantir/blueprint/blob/274ef9fb153e1879551e1e9aaddd2ca3f03c775c/packages/core/src/components/tabs/tabs.tsx#L295), [SCSS](https://github.com/palantir/blueprint/blob/274ef9fb153e1879551e1e9aaddd2ca3f03c775c/packages/core/src/components/tabs/_tabs.scss#L145-L147)).